### PR TITLE
Add initial demo instructions - before automation

### PR DESCRIPTION
**Pull request by kylienorwood**
This document simply explains how to run each entity in a separate terminal window, but demonstrates how each is working together. Down the line this will be automated, but for now it gives a good understanding of what is happening and how everything is related to each other.

This may not be something we actually want to merge in since it's quite temporary, but it might help others better see the relationship between everything, so I thought I'd write something up.

### CODE REVIEW

**Comment by rileyalankirk** [comment]
Looks great!

File: docs/inital_demo_instructions.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+1. Open a fifth (and final) terminal window and navigate to the **Kylie** directory.  Execute `cd capstone2021`, complete the *Note* steps, and then enter the appropriate branch: `git checkout dashboard_documentation` (not the most accurate name here anymore).
+2. Navigate to the location of the appropriate file: `cd src/c21server/dashboard`.
+3. Run the file: `python3 dashboard_server.py`. 
+4. You should see: `* Running on http://[IP_ADDRESS]:5000/ (Press CTRL+C to quit)`.
+5. Paste `[IP_ADDRESS]:5000` in your browser. 
+6. We have two endpoints - `data` and `dashboard`. You will most likely have an error when just pasting the original address in your browser. Decide which endpoint you'd like to view and paste either `[IP_ADDRESS]:5000/data` or `[IP_ADDRESS]:5000/dashboard`. 
+
+Periodically refresh and see the updates on the dashboard! 
+
+### Part 4
```
Comment (jonahb13) [inline docs/inital_demo_instructions.md]: I believe that this is Part 3 and also should be a secondary header (##, just for consistency with headers for the parts), but overall looks great!

**Comment by Jarod-Frekot** [comment]
You can have the 5 different terminals in virtual environments and they work fine. I tested it earlier.

**Comment by kylienorwood** [comment]
We can probably close this pull request early next week, since we have the Vagrant aspect working and we are automated. For now, we can leave it until after the sprint review on Friday, because we plan to do a demo similar to this process. Afterwards, I will close it.

**Pull request closed without merging**